### PR TITLE
Accessability improvements

### PR DESCRIPTION
**Pull request by deti**
There are few changes

1. Fix merged from the previous branch which already reviewed
1. Replaced official Readme with our CKEditor notes.
1. I made Readme file **rst** formatted to mirror http://jenkins.pstattest.com/job/pstat_master_sphinx_docs/Sphinx_Docs/front_end_guidelines/ckeditor.html
1. Also I made `pstat` branch default on the repo to be able easily find right place for editing

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+********************************
+
+.. _ckeditor-pages-using-ckeditor:
+
+Pages Using CKEditor
+====================
+
+Currently, CKEditor is enabled as the editor for the following pages:
+
+* Document clean
```
Comment (kylegibson-rldatix) [inline README.rst:23]: This isn't the only page that CKeditor is enabled for. It should be active on the normal document edit and override pages too. Please verify.

Comment (bocribbz) [reply README.rst:23]: bump

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+If you attempt to load a plugin that does not exist, a script error will occur
+in all CKEditor selenium tests.
+
+
+Useful links
+============
+
+* `CKEditor 4 docs
+  <https://ckeditor.com/docs/ckeditor4/latest/index.html>`_
```
Comment (bocribbz) [inline README.rst:191]: yay up to date documentation!

File: plugins/tabletools/dialogs/tableCell.js [hunk 3]
```diff
@@ -332,97 +332,6 @@ CKEDITOR.dialog.add( 'cellProperties', function( editor ) {
...
-								this.getElement().getParent().setStyle( 'vertical-align', 'bottom' );
-							},
-							onClick: function() {
-								editor.getColorFromDialog( function( color ) {
-									if ( color )
-										this.getDialog().getContentElement( 'info', 'borderColor' ).setValue( color );
-									this.focus();
-								}, this );
-							}
-						} : spacer ]
```
Comment (bocribbz) [inline plugins/tabletools/dialogs/tableCell.js:425]: is this section removed from merging up with main repo?

**Review by bocribbz — APPROVED** [review]
👍

File: README.rst (hunk 1 above)
Comment (deti) [reply README.rst:23]: Yes. They are if CKEditor is enabled.

These templates contain CKEditor
- ./pstat/templates/document_control/document_admin_override_ckeditor.haml ✔️ 
- ./pstat/templates/document_control/document_edit_ckeditor.haml ✔️ 
- ./pstat/templates/document_import/clean_import_ckeditor.haml
- ./pstat/templates/editor_sandbox/editor_sandbox.haml 
- ./pstat/templates/import_translation/test_cleaning_upload.haml

File: plugins/tabletools/dialogs/tableCell.js (hunk 3 above)
Comment (deti) [reply plugins/tabletools/dialogs/tableCell.js:425]: This is actual fix for the request. We have to prohibit color changes. So, we don't elements and functionality for them.

File: README.rst (hunk 2 above)
Comment (deti) [reply README.rst:191]: \o/

**Pull request merged**